### PR TITLE
Fix broken engage ROS link

### PR DESCRIPTION
**Pull request by britneywwc**
## Done

- fix broken engage ROS link
- previously `/engage/Deploying-ros-applications-Developer-guide`, replaced with `/engage/deploying-ros-applications-developer-guide`
- apply linting and formatting to affected files

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes [WD-13036](https://warthogs.atlassian.net/browse/WD-13036)

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-13036]: https://warthogs.atlassian.net/browse/WD-13036?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/robotics/index.html [hunk 1]
```diff
@@ -295,227 +380,245 @@ <h2>Robotics software updates</h2>
...
+            }}
+            <h3 class="p-heading-icon__title p-heading--4">Tutorial</h3>
+          </div>
         </div>
+        <ul class="p-list">
+          <li class="p-list__item">
+            <a href="https://snapcraft.io/docs/robotics">Containerisation for robotics applications</a>
+          </li>
+          <li class="p-list__item">
+            <a href="/engage/deploying-ros-applications-developer-guide">Deploying ROS applications with snaps</a>
```
Comment (britneywwc) [inline templates/robotics/index.html:498]: Change here

File: templates/robotics/what-is-ros.html [hunk 2]
```diff
@@ -1,234 +1,251 @@
...
+          ROS is led by Open Robotics, similar to how Canonical supports Ubuntu; Open Robotics steers the ship but it is driven by the community.
+        </p>
+        <p>
+          As the robotics industry grows, companies and developers continue turning to Open Robotics and Canonical to help make their vision a reality.
+        </p>
+        <p>
+          <a class="p-button--positive" href="https://wiki.ros.org/ROS/Tutorials">Get started with ROS</a>
+        </p>
+        <p>
+          <a href="/engage/deploying-ros-applications-developer-guide">
```
Comment (britneywwc) [inline templates/robotics/what-is-ros.html:37]: Change here

File: templates/robotics/index.html [hunk 3]
```diff
@@ -2,149 +2,155 @@
```
Comment (britneywwc) [inline templates/robotics/index.html:2]: Big diff due to linting, I commented out the parts where changes were made

File: templates/robotics/what-is-ros.html [hunk 4]
```diff
@@ -1,234 +1,251 @@
 {% extends "robotics/base_robotics.html" %}
```
Comment (britneywwc) [inline templates/robotics/what-is-ros.html:1]: Big diff due to linting, I commented out the parts where changes were made

File: templates/robotics/index.html [hunk 5]
```diff
@@ -167,126 +173,205 @@ <h2 class="p-muted-heading">Building the future with</h2>
...
-  <div class="u-fixed-width u-align--center">
-    <h2 class="u-sv3">Bringing Ubuntu's worldwide infrastructure to your robot</h2>
+  <section class="p-strip u-no-padding--bottom">
+    <div class="u-fixed-width u-align--center">
+      <h2 class="u-sv3">Bringing Ubuntu's worldwide infrastructure to your robot</h2>
 
-    <div class="u-hide--large">
-      {{ image (
+      <div class="u-hide--large">
+        {{ image (
```
Comment (akbarkz) [inline templates/robotics/index.html:192]: please remove space between `image` and the bracket and re-format

File: templates/robotics/index.html [hunk 6]
```diff
@@ -167,126 +173,205 @@ <h2 class="p-muted-heading">Building the future with</h2>
...
-      </ul>
-      <p>
-        <a href="/core/features/ota-updates" class="p-button-positive">Learn more about OTA updates for Ubuntu Core&nbsp;&rsaquo;</a>
-      </p>
-    </div>
-    <div class="col-6 u-hide--small u-hide--medium u-align--center u-vertically-center">
-      {{
+      <div class="col-6 u-hide--small u-hide--medium u-align--center u-vertically-center">
+        {{
         image (
```
Comment (akbarkz) [inline templates/robotics/index.html:375]: same here

File: templates/robotics/index.html [hunk 7]
```diff
@@ -295,227 +380,245 @@ <h2>Robotics software updates</h2>
...
-    </div>
-    <div class="col-4 p-divider__block">
-      <div class="p-heading-icon--muted">
-        <div class="p-heading-icon__header">
-          {{ 
+      <div class="col-4 p-divider__block">
+        <div class="p-heading-icon--muted">
+          <div class="p-heading-icon__header">
+            {{
             image (
```
Comment (akbarkz) [inline templates/robotics/index.html:480]: and here

**Review by akbarkz — APPROVED** [review]

**Pull request merged**